transactions/src/main/resources/org/jboss/as/txn/subsystem/LocalDescriptions.properties

```diff
@@ -4,43 +4,39 @@ transactions.remove=Removes the transaction subsystem
 
 transactions.configuration=various environment configuration. It could refer to core, recovery, coordinator environment and to object store configs.
 
-transactions.node-identifier=Used to set the node identifier on the core environment.
+transactions.node-identifier=Used to set the node identifier on the core environment. Each Xid that Narayana creates will have this identifier encoded within it and ensures Narayana will only recover branches which match the specified identifier. It is imperative that this identifier is unique between WildFly instances which share either an object store or access common resource managers.
```

Comment:
If possible we should use more neutral terms instead of project names like Narayana and WildFly.